src/components/toggle/toggle.svelte

```diff
@@ -241,6 +241,7 @@
       .thumb {
         --thumb-offset: var(--checked-thumb-offset);
         color: var(--checked-color);
+        --thumb-disabled-color: var(--leo-color-schemes-on-primary);
```

Comment:
This breaks the themeability of the toggle:
![SCR-20241126-jnjx](https://github.com/user-attachments/assets/003c1973-3c01-4272-926b-f2884d28632d)

Reply:
Ugh :(

Reply:
@fallaciousreasoning @AlanBreck  shouldn't `--leo-color-schemes-on-primary` change depending on the color that is chosen as primary? Or how should I do that? 


Btw I just realized this is also broken on the working version? If I customize the color there it doesn't show it on the thumb
https://nala.bravesoftware.com/?path=/docs/components-toggle--docs&globals=theme:dark

Reply:
![Uploading image.png…]()

Reply:
Ok, I think I fixed it now.

I can change the color to whatever I want and it doesn't break on the default colors

<img width="334" alt="image" src="https://github.com/user-attachments/assets/75859d60-424b-4fb4-97fa-8a7260efa93d">

<img width="336" alt="image" src="https://github.com/user-attachments/assets/a5e5d860-14bf-464f-9d57-268c3cec7496">